src/cli.yaml

```diff
@@ -278,6 +278,11 @@ subcommands:
             long: tsv
             value_name: TSV_FILE_PATH
             help: Add a TSV file that contains one or multiple custom values for each sample for the oncoprint. First column has to be the sample name, followed by one or more columns with custom values. Make sure you include one row for each given sample.
+        - threads:
+            long: threads
+            required: false
+            default_value: "8"
```

Comment:
I think the default should be `0`, since rayon uses the number of CPUs if `num_cpus == 0` *or* the environment variable `RAYON_NUM_THREADS` if set.

Reply:
On a different note, I'd strongly recommend switching from clap+yaml to structopt, since that has way less boilerplate code (i.e. no parsing steps etc)

Reply:
> I think the default should be `0`, since rayon uses the number of CPUs if `num_cpus == 0` _or_ the environment variable `RAYON_NUM_THREADS` if set.

Sounds good. I was't to sure about the `8` either.

> On a different note, I'd strongly recommend switching from clap+yaml to structopt, since that has way less boilerplate code (i.e. no parsing steps etc)

I will have a look into structop but that seems to be something for another PR  😉.